src/test/java/com/exasol/adapter/document/DynamodbVsExasolTestDatabaseBuilder.java

```diff
@@ -0,0 +1,67 @@
+package com.exasol.adapter.document;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.sql.SQLException;
+import java.util.concurrent.TimeoutException;
+
+import com.exasol.bucketfs.BucketAccessException;
+
+/**
+ * DynamoDB specific test database builder.
+ */
+public class DynamodbVsExasolTestDatabaseBuilder extends ExasolTestDatabaseBuilder {
+    private static final String VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION = "document-virtual-schema-dist-0.3.0-dynamodb-0.5.0.jar";
+    private static final Path PATH_TO_VIRTUAL_SCHEMAS_JAR = Path.of("target", VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+
+    public DynamodbVsExasolTestDatabaseBuilder(final ExasolTestInterface testInterface)
+            throws SQLException, IOException {
+        super(testInterface);
+    }
+
+    /**
+     * Uploads the dynamodb adapter jar to the exasol test container.
+     */
+    public void uploadDynamodbAdapterJar() throws InterruptedException, BucketAccessException, TimeoutException {
+        this.testInterface.uploadFileToBucketfs(PATH_TO_VIRTUAL_SCHEMAS_JAR, VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+        uploadJacocoAgent();
+    }
+
+    public void createUdf() throws SQLException {// TODO make DynamoDB independent
+        final StringBuilder statementBuilder = new StringBuilder("CREATE OR REPLACE JAVA SET SCRIPT ")
+                .append(ADAPTER_SCHEMA).append(".").append(UdfRequestDispatcher.UDF_PREFIX + "DYNAMO_DB").append("(")
+                .append(AbstractDataLoaderUdf.PARAMETER_DOCUMENT_FETCHER).append(" VARCHAR(2000000), ")
+                .append(AbstractDataLoaderUdf.PARAMETER_REMOTE_TABLE_QUERY).append(" VARCHAR(2000000), ")
+                .append(AbstractDataLoaderUdf.PARAMETER_CONNECTION_NAME).append(" VARCHAR(500)) EMITS(...) AS\n");
+        addDebuggerOptions(statementBuilder, true);
```

Comment:
It's usually not a good idea to send a builder inside and make some magic in a black box. It's better to return a String and append it ouside

Reply:
I totally agree, the issue is just, that `addDebuggerOptions` is part of the document-vs so I would have to create another release for that...

Reply:
Create a ticket and change this behavior next time you touch that common module

Reply:
ok. https://github.com/exasol/virtual-schema-common-document/issues/17